webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const webpack = require('webpack');
+
+module.exports = {
+    entry: './src/index.js',
+    module: {
+        rules: [{
+            test: /\.(js|jsx)$/,
+            exclude: /node_modules/,
+            use: ['babel-loader']
+        },
+        {
+            test: /\.(css|scss)$/,
+            use: [
+                'style-loader',
+                'css-loader',
+                'sass-loader'
+          ]  
+        },
+        {
+            test: /\.(jpg|jpeg|mp3|gif|png|svg)$/,
+            loaders: ['file-loader']
+        }]
+    },
+      resolve: {
+        extensions: ['*', '.js', '.jsx']
+      },
+    output: {
+      path: __dirname + '/public',
```

Comment:
Can you separate the bundled file into another folder `e.g. Build`, it should a be a better convention because the public folder is mainly for rendering static files

Reply:
Done